stable/appmesh-controller/README.md

```diff
@@ -46,6 +46,38 @@ The [configuration](#configuration) section lists the parameters that can be con
 
 ## Upgrade
 
+This section will assist you in upgrading the appmesh-controller from <=v0.5.0 version to >=v1.0.0 version.
+
+You can either build new CRDs from scratch or migrate existing CRDs to the new schema. Please refer to the documentation [here for the new API spec](https://aws.github.io/aws-app-mesh-controller-for-k8s/reference/api_spec/). Also, you can find several examples [here](https://github.com/aws/aws-app-mesh-examples/tree/master/walkthroughs) with v1beta2 spec to help you get started.
+
+Starting v1.0.0, Mesh resource supports namespaceSelectors, where you can either select namespace based on labels (recommended option) or select all namespaces. To select a namespace in a Mesh, you will need to define `namespaceSelector`:
+
+```
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: Mesh
+metadata:
+  name: <mesh-name>
+spec:
+  namespaceSelector:
+    matchLabels:
+      mesh: <mesh-name> // any string value
+```
+
+Note: If you set `namespaceSelector: {}`, mesh will select all the namespace in your cluster.
+
+In the namespace spec, you will need to add a label `mesh: <mesh-name>`. Here's a sample namespace spec:
```

Comment:
Can we add a comment for when the Mesh is selecting on all the namespaces?

Reply:
Absolutely. Added a note that label is a no-op when selecting app